### PR TITLE
PMM-7427 Fix disk size parser.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: utils/convertors/convertors.go [hunk 1]
```diff
@@ -24,8 +24,13 @@ import (
 
 // StrToBytes converts string of bytes to integer.
 func StrToBytes(s string) int64 {
+	multiplier := int64(1)
+	if strings.HasSuffix(s, "G") {
```
Comment (jprukner) [inline utils/convertors/convertors.go:28]: Kubernetes supports "Gi" suffix, too. If we make sure we use "G" always, it's fine.

**Review by jprukner — APPROVED** [review]

File: service/k8sclient/k8client.go [hunk 2]
```diff
@@ -593,6 +593,12 @@ func (c *K8Client) ListPSMDBClusters(ctx context.Context) ([]PSMDBCluster, error
 
 // CreatePSMDBCluster creates percona server for mongodb cluster with provided parameters.
 func (c *K8Client) CreatePSMDBCluster(ctx context.Context, params *PSMDBParams) error {
+	affinity := &psmdb.PodAffinity{
+		TopologyKey: pointer.ToString(psmdb.AffinityOff),
+	}
+	if isMinikube, err := c.isMinikube(ctx); err == nil && !isMinikube {
```
Comment (jprukner) [inline service/k8sclient/k8client.go:599]: It just came to my mind this morning. Wouldn't it be better to check for single node cluster, or cluster where only single node could be used to deploy apps?

Comment (askomorokhov) [reply service/k8sclient/k8client.go:599]: Such check may not work  https://minikube.sigs.k8s.io/docs/tutorials/multi_node/

**Review by askomorokhov — APPROVED** [review]

**Comment by AlekSi** [comment]
There are conflicts

**Comment by jprukner** [comment]
> There are conflicts

@AlekSi What do you mean?

```sh
$ pwd
/home/jan/go/src/github.com/percona-platform/dbaas-controller

$ git checkout PMM-7247-fix-disk-size-value
Branch 'PMM-7247-fix-disk-size-value' set up to track remote branch 'PMM-7247-fix-disk-size-value' from 'origin'.
Switched to a new branch 'PMM-7247-fix-disk-size-value'

$ git merge PMM-5988-dbaas-add-progress-bar
PMM-5988-dbaas-add-progress-bar     PMM-7144-reject-same-cluster-name   PMM-7247-fix-disk-size-value        
$ git merge PMM-5988-dbaas-add-progress-bar 
Auto-merging service/k8sclient/k8client.go
Auto-merging go.sum
Auto-merging go.mod
Merge made by the 'recursive' strategy.
 go.mod                                               |   2 +-
 go.sum                                               |   2 ++
 service/cluster/psmdb_cluster.go                     |   4 +++-
 service/cluster/xtra_db_cluster.go                   |   5 +++--
 service/k8sclient/internal/psmdb/psmdb_types.go      |   7 +++++++
 service/k8sclient/internal/psmdb/psmdb_types_test.go |   4 +++-
 service/k8sclient/k8client.go                        |  87 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++----------------------
 service/k8sclient/k8client_test.go                   | 112 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++-----------------------------------------
 8 files changed, 155 insertions(+), 68 deletions(-)
```

**Comment by AlekSi** [comment]
image

**Comment by BupycHuk** [comment]
@AlekSi @jprukner I'll fix it

**Review by jprukner — APPROVED** [review]

**Pull request merged**